firmware/controllers/map_averaging.cpp

```diff
@@ -193,6 +213,17 @@ static void mapAveragingCallback(trigger_event_e ckpEventType,
 		return;
 	}
 
+	if (boardConfiguration->mapMinBufferLength != mapMinBufferLength) {
+		// check range
+		mapMinBufferLength = maxI(minI(boardConfiguration->mapMinBufferLength, MAX_MAP_BUFFER_LENGTH), 0);
+		// reset index
+		averagedMapBufIdx = 0;
+		// fill with maximum values
+		for (int i = 0; i < MAX_MAP_BUFFER_LENGTH; i++) {
+			averagedMapRunningBuffer[i] = CONFIG(mapErrorDetectionTooHigh);
```

Comment:
technically could be just SOME_HUGE_VALUE just to make code simpler by reducing dependency on "CONFIG(mapErrorDetectionTooHigh)"